src/plugins/MediaManager.js

```diff
@@ -130,9 +133,10 @@ module.exports = class MediaManager {
             log.debug('Client audio stream ready');
             this.audioStreaming = true;
             this.localAudioStream = mediaStream;
-            this.addAudioStream(mediaStream);
+            return this.addAudioStream(mediaStream);
```

Comment:
if addAudioStream fail the error will be uncaught. `this.instance.renegotiateWebRTCConnection();` call renegotiateWebRTCConnection wihch do `this.peerConnection.createOffer` and createOffer return a promise.

I think `addAudioStream` need to return `this.instance.renegotiateWebRTCConnection();` or a Promise.reject() and this line should be prefix by an await.

Reply:
I was mostly focused on errors related to permissions or availability of input devices but you're right I should cover this case too. Thanks !

Reply:
Is it OK for you now ?